login.html

```diff
@@ -1,9 +1,10 @@
 <html>
-<meta name="google-signin-client_id" content="XXX.apps.googleusercontent.com">
+<meta name="google-signin-client_id" content="208463168480-m5fronlc3ic98elbicq7gmglst8trdkg.apps.googleusercontent.com">
```

Comment:
@Vedant1202  You should probably remove your client ID :)

Reply:
yes, done!